internal/k8s/selfregister.go

```diff
@@ -128,6 +128,28 @@ func (a *AdmissionWebhookRegisterClient) Register(ctx context.Context, c *config
 						},
 					},
 				},
+				NamespaceSelector: &metav1.LabelSelector{
+					MatchExpressions: []metav1.LabelSelectorRequirement{
+						{
+							Key:      "name",
```

Comment:
can't we just use one label selector? which would be "kubernetes.io/metadata.name", instead of adding two for same values and doing AND operator btw them.

Reply:
Sure